docs/Commands.md

```diff
@@ -19,3 +19,20 @@ Add Command `Hello chat :)` with the trigger `/hello`. Now typing `/hello` in ch
   Example: Add Command `Have a {1+} day!` with trigger `/day`. Now typing `/day very super nice` will send `Have a very super nice day!`
 
 - You can use `{{1}` if you want to send `{1}` literally.
+
+Placeholders below are available in nightly:
+
+- `{channel}` can be used to insert the name of the current channel
+
+    Example: Add Command `/openurl https://twitch.tv/{channel}` with trigger `/openchannel`. Now typing `/openchannel` will open the current channel in your browser.
+
+- `{channelid}` can be used to insert the Twitch account ID of the owner of current channel
+- `{myid}` can be used to insert the Twitch account ID of the currently selected account
+- `{myname}` can be used to insert the Twitch username of the currently selected account
+- `{game}` can be used to insert the game set in the current channel
+
+    Example: Add Command `I'm currently playing {game}` with trigger `/game`. Now typing `/game` will send a message like `I'm currently playing World of Warcraft`.
+
+- `{title}` can be used to insert the title set in the current channel
```

Comment:
I feel like `can be used to insert` is not the right term to use here, perhaps `can replace` or just `replaces` should work better. CC pajlada, @felanbird

Reply:
I actually think insert might be correct here, purely because that's exactly what you are doing, you are inserting the current channel into a command.
![image](https://user-images.githubusercontent.com/41973452/132861483-9d4c7edc-bac8-4b8c-a967-a69c64a81d70.png)